palette/src/encoding/pixel/mod.rs

```diff
@@ -52,23 +52,18 @@ mod raw;
 ///     let buffer = [0.1, 0.2, 0.3, 0.4];
 ///     let color = MyCmyk::from_raw(&buffer);
 ///
-///     assert_eq!(
-///         color,
-///         &MyCmyk {
-///             cyan: 0.1,
-///             magenta: 0.2,
-///             yellow: 0.3,
-///             key: 0.4,
-///         }
-///     );
+///     assert_relative_eq!(color.cyan, 0.1);
+///     assert_relative_eq!(color.magenta, 0.2);
+///     assert_relative_eq!(color.yellow, 0.3);
+///     assert_relative_eq!(color.key, 0.4);
```

Comment:
This one and the other one in the same file can also use `assert_eq`. They will just copy the data.